x-pack/test/api_integration/apis/ml/data_frame_analytics/post.ts

```diff
@@ -0,0 +1,158 @@
+        training_percent: 20,
+      },
+    },
+    analyzed_fields: {
+      includes: [],
+      excludes: [],
+    },
+    model_memory_limit: '60mb',
+    allow_lazy_start: false, // default value
+    max_num_threads: 1, // default value
+  };
+
+  const testJobConfigs: Array<DeepPartial<DataFrameAnalyticsConfig>> = [
+    {
+      id: `${jobId}_0`,
+      description: 'Test start and stop for analytics',
+      dest: {
+        index: generateDestinationIndex(`${jobId}_0`),
+        results_field: 'ml',
+      },
+      ...commonJobConfig,
+    },
+  ];
+
+  async function createJobs(mockJobConfigs: Array<DeepPartial<DataFrameAnalyticsConfig>>) {
+    for (const jobConfig of mockJobConfigs) {
+      await ml.api.createDataFrameAnalyticsJob(jobConfig as DataFrameAnalyticsConfig);
+    }
+  }
+
+  describe('POST data_frame/analytics', () => {
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/bm_classification');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await createJobs(testJobConfigs);
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    describe('StartDataFrameAnalyticsJob', () => {
+      it('should start analytics job for specified id if job exists', async () => {
+        const analyticsId = `${jobId}_0`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${analyticsId}/_start`)
+          .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(200);
+
+        expect(body).not.to.be(undefined);
+        expect(body.acknowledged).to.be(true);
+        expect(body.node).not.to.be('');
+      });
+
+      it('should show 404 error if job does not exist', async () => {
+        const id = `${jobId}_invalid`;
+        const message = `No known job with id '${id}'`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${id}/_start`)
+          .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(404);
+
+        expect(body.error).to.eql('Not Found');
+        expect(body.message).to.eql(message);
+      });
+
+      it('should not allow to start analytics job for unauthorized user', async () => {
+        const analyticsId = `${jobId}_0`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${analyticsId}/_start`)
+          .auth(USER.ML_UNAUTHORIZED, ml.securityCommon.getPasswordForUser(USER.ML_UNAUTHORIZED))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(403);
+
+        expect(body.error).to.eql('Forbidden');
+        expect(body.message).to.eql('Forbidden');
+      });
+    });
+
+    describe('StopsDataFrameAnalyticsJob', () => {
+      it('should stop analytics job for specified id when job exists', async () => {
+        const analyticsId = `${jobId}_0`;
```

Comment:
I think it's not ideal to use the same job as for the `start` test. Even if we don't wait for the job to finish as suggested above, it might have already finished so there's nothing to stop.
I'd suggest to use a separate job, configure it to be "slow running" and create it in the `before` of this `stop` suite, such that we can kind of guarantee that it's still open (similar to what we do in the `stop_transforms` API test).